test: increase size tracking threshold to 5K / 5%

alxhub opened the pull request:
This commit increases the threshold of payload size tracking tests from 500 bytes to 5,000 bytes, and from 1% to 5%. This is done to minimize merge conflicts while still catching real regressions.

josephperrott reviewed (approved):
LGTM

devversion commented on .circleci/config.yml:
```diff
@@ -339,7 +339,8 @@ jobs:
         # Check the bundle sizes.
-      - run: yarn --cwd aio payload-size
+      # TODO: size tracking is disabled until we figure out a better option
+      # - run: yarn --cwd aio payload-size
```
What about AIO local?

Also I think we should much rather continue doing what we have now- until we have an alternative. We can bump the threshold from 500b to e.g. 3000b to ease updates- but avoid major regressions.

alxhub committed:
test: increase size tracking threshold to 5K / 5%

This commit increases the threshold of payload size tracking tests from
500 bytes to 5,000 bytes, and from 1% to 5%. This is done to minimize merge
conflicts while still catching real regressions.

pkozlowski-opensource commented:
This PR was merged into the repository by commit c4d2a55c44437993166213caafeb9eb104090a90.
